tyrian/js/src/main/scala/tyrian/TyrianRoutedAppF.scala

```diff
@@ -0,0 +1,45 @@
+package tyrian
+
+import cats.effect.kernel.Async
+import cats.effect.kernel.Resource
+import org.scalajs.dom.Element
+import org.scalajs.dom.document
+import tyrian.runtime.TyrianRuntime
+
+import scala.scalajs.js.annotation._
+
+/** The TyrianApp trait can be extended to conveniently prompt you for all the methods needed for a Tyrian app, as well
+  * as providing a number of standard app launching methods.
+  */
+trait TyrianRoutedAppF[F[_]: Async, Msg, Model] extends TyrianAppF[F, Msg, Model]:
+
+  def hashResultToMessage: Navigation.Result => Msg
+  def hashChangeToMessage: Navigation.Result.HashChange => Msg
```

Comment:
I'd rather make it clear to the user that these functions are used to actually spit a message **everytime** the app initializes and the hash/URL changes. Maybe it's worth adding a paragraph in a new docs section for `Routing`. Anyway, they'll be forced to deal with the messages in the `update` function so not critical in the end.

Reply:
I'm hoping these two go away / are replaced. They're just a hangover from `Navigation`. :crossed_fingers: